packages/roosterjs-react-command-bar/lib/components/RoosterCommandBar.tsx

```diff
@@ -1,38 +1,30 @@
 import * as React from 'react';
 import { Async, css } from 'office-ui-fabric-react/lib/Utilities';
-// TODO-1 import { CommandBar, ICommandBarItemProps } from '@uifabric/experiments/lib/CommandBar';
```

Comment:
Not planning to use the experimental command bar anymore?

Reply:
nope, even the owner doesn't know when it'll ship